tests/test_ic_od_to_kvp_wrapper/test_od_as_kvp.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+
+from tests.test_fixtures import DetectionTestFixtures
+from vision_datasets.common.constants import DatasetTypes
+from vision_datasets.image_object_detection import DetectionAsKeyValuePairDataset
+from vision_datasets.key_value_pair.manifest import KeyValuePairLabelManifest
+
+
+class TestDetectionAsKeyValuePairDataset(unittest.TestCase):
+    def test_detection_to_kvp(self):
+        sample_detection_dataset, _ = DetectionTestFixtures.create_an_od_dataset()
+        kvp_dataset = DetectionAsKeyValuePairDataset(sample_detection_dataset)
+
+        self.assertIsInstance(kvp_dataset, DetectionAsKeyValuePairDataset)
+        self.assertEqual(kvp_dataset.dataset_info.type, DatasetTypes.KEY_VALUE_PAIR)
+        self.assertIn("name", kvp_dataset.dataset_info.schema)
+        self.assertIn("description", kvp_dataset.dataset_info.schema)
+        self.assertIn("fieldSchema", kvp_dataset.dataset_info.schema)
+
+        self.assertEqual(kvp_dataset.dataset_info.schema["fieldSchema"],
+                         {'detectedObjects': {'type': 'array', 'description': 'Objects in the image of the specified classes, with bounding boxes',
+                                              'items': {'type': 'string', 'description': 'Class name of the object',
+                                                        'classes': {'1-class': {},
+                                                                    '2-class': {},
+                                                                    '3-class': {},
+                                                                    '4-class': {}},
+                                                        'includeGrounding': True}}})
+
```

Comment:
could you add a single-class OD test case?

Reply:
added